target/classes/META-INF/maven/com.Maven/java-bootcamp/pom.xml

```diff
@@ -0,0 +1,32 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
The target folder shouldn't be pushed as part of the project in git

Reply:
Unnecessary folder was removed from this project.